include/kvm_websocket.hpp

```diff
@@ -20,7 +20,7 @@ class KvmSession
         conn(conn), doingWrite(false), hostSocket(conn.get_io_context())
     {
         boost::asio::ip::tcp::endpoint endpoint(
-            boost::asio::ip::make_address("::1"), 5900);
+            boost::asio::ip::make_address("127.0.0.1"), 5900);
```

Comment:
Why change this?

Reply:
It matches upstream: 
https://github.com/ibm-openbmc/bmcweb/commit/b8c7eb23f9b598274792fe3b8d178d1f2e4cc6b0